### ISSUE TITLE
SCP-294 does not retrieve blood when provided with the name of a mob

### ISSUE THREAD

**Issue by Exxion** [d3athrow/vgstation13#8619]
This is a heinous oversight

**Comment by xpcybic** [comment]
```
>ask for "cup of grey shit"
>receive liquefied mixture of internal organs and bodily fluids
>somewhere down the hall, assistant mcgreyshit suddenly collapses
```

Alternatively

```
>ask for "cup of grey shit"
>receive cup of yahir's poo
```

**Comment by Zth--** [comment]
This gave me a good idea. I'm gonna work on this after another project I have in mind

**Comment by Killette2** [comment]
tfw drinking a cup of yahir's poo to soothe your addiction for the next few hours

**Comment by jernaumorat** [comment]
Where is it on the map? I've been looking around since it was merged and haven't been able to find it.

**Comment by PJB3005** [comment]
@Vermilingua nowhere, it's admin spawn only.

**Comment by jernaumorat** [comment]
Right, makes sense.

**Comment by uazuaz** [comment]
does the SCP-294 provide liquids only? or it could provide any item?

**Comment by Zth--** [comment]
On the next update it will dispense cups of paper, you'll be able to get a cup of any mob and I'll try to add it to xenoarch but I dont know how.

Yes, only liquids.

**Comment by Imagebard** [comment]
Liquids only.

**Comment by Zth--** [comment]
I think I can use ghettochem to get the reagents of items that can be grinded, but I'll check that later

**Comment by SECBATON-GRIFFON** [comment]
a PR i did made it do that, outdated issue

**Comment by Exxion** [comment]
Resolved by #31830
